Codeinwp/otter-blocks
# Improve Marketing Integration info & bug fix

Closes #1290

### Summary


🖊️  Improved the Marketing Action explanations.
🐛 Fixed a bug related to the default action.

### Screenshots 

![image](https://github.com/Codeinwp/otter-blocks/assets/17597852/216ff038-787e-4674-b698-4a08609f050c)

![image](https://github.com/Codeinwp/otter-blocks/assets/17597852/603b6b21-8a5b-40bc-b637-da6ecd999f71)

![image](https://github.com/Codeinwp/otter-blocks/assets/17597852/bbd8cc6c-2a56-4535-9169-3646fd587d0c)


----

### Test instructions


⚠️ This is tied to the `development` branch. When `next` will be merged, I will update this.

⚠️ You will need a Mailchimp and Sendinblue account.

1. Make a Form
2. Fill in the Marketing integration options.
3. Ensure that they work as expected.



---- 

### Checklist before the final review
